src/main/java/duke/PersistentStorageHandler.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import java.io.*;
```

Comment:
Shouldn't you import the specific library you used rather than the * sign.

Reply:
I used[ this tutorial](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class) to help me configure Intellij to import my classes easier without converting them into wildcards!